projects/demo/src/modules/components/line-days-chart/examples/2/pipe.ts

```diff
@@ -15,6 +15,7 @@ function even<T>(array: readonly T[]): readonly T[] {
 }
 
 @Pipe({
+    standalone: true,
```

Comment:
Maybe we should move this class to `index.ts` so we can use our regular dynamic examples?

Reply:
fixed